mamedov_timur/graph.cpp

```diff
@@ -0,0 +1,85 @@
+#include <iostream>
+#include <vector>
+
+class Graph {
+ public:
+  using VertexId = int;
+  using EdgeId = int;
+  void add_vertex() { vertexes_.emplace_back(get_new_vertex_id()); }
+
+  void add_edge(VertexId from_vertex_id, VertexId to_vertex_id) {
+    edges_.emplace_back(get_new_edge_id(), from_vertex_id, to_vertex_id);
```

Comment:
Похоже что у тебя здесь баг
Добавь проверки на существование вершин
```cpp
#include <cassert>
```
```suggestion
    assert(has_vertex(from_vertex_id));
    assert(has_vertex(to_vertex_id));
    edges_.emplace_back(get_new_edge_id(), from_vertex_id, to_vertex_id);
```
```cpp
bool has_vertex(VertexId id) const;
```